tests/snuba/api/endpoints/test_organization_tagkey_values.py

```diff
@@ -152,3 +153,25 @@ def test_array_column(self):
 
     def test_no_projects(self):
         self.run_test("fruit", expected=[])
+
+    def test_transaction_tags(self):
+        data = load_data("transaction")
+        data["timestamp"] = iso_format(before_now(minutes=1))
+        data["start_timestamp"] = iso_format(before_now(minutes=1, seconds=5))
+        self.store_event(
+            data, project_id=self.project.id,
+        )
+        transaction = data.copy()
+        transaction["timestamp"] = iso_format(before_now(seconds=30))
+        transaction["start_timestamp"] = iso_format(before_now(seconds=35))
+        transaction["contexts"]["trace"]["status"] = "invalid_argument"
+        self.store_event(
+            transaction, project_id=self.project.id,
+        )
+        self.run_test("transaction.status", expected=[("invalid_argument", 1), ("ok", 1)])
```

Comment:
Could we include a test for partial hits on the status values? Perhaps looking for the `inval` text fragment?